app/package.json

```diff
@@ -22,6 +22,7 @@
     "chalk": "^2.3.0",
     "classnames": "^2.2.5",
     "codemirror": "^5.31.0",
+    "codemirror-mode-elixir": "^1.1.1",
```

Comment:
Is there a way to accomplish Elixer highlighting without adding an additional dependency?

Reply:
According to CodeMirror’s contributing guide, they [don’t accept new modes](https://github.com/codemirror/CodeMirror/blob/master/CONTRIBUTING.md#contributing-code) in the `codemirror` package, and while you could re-implement [the mode itself](https://github.com/optick/codemirror-mode-elixir/blob/master/index.js), it’s nearly 400 lines and there doesn’t seem to be a clear benefit to doing so.

Reply:
@joaovitoras Could you please change this to an exact version (i.e. drop the leading caret). We don't want this package to change from underneath us without an explicit review of the changes.